terraform-modules/cur-setup-source/main.tf

```
@@ -121,6 +125,16 @@ data "aws_iam_policy_document" "bucket_policy" {
       aws_s3_bucket.this.arn,
       "${aws_s3_bucket.this.arn}/*",
     ]
+    condition {
+      test     = "StringEquals"
+      values   = ["arn:${data.aws_partition.current.partition}:cur:${data.aws_region.current.region}:${data.aws_caller_identity.current.account_id}:definition/*"]
+      variable = "aws:SourceArn"
+    }
+    condition {
+      test     = "StringEquals"
+      values   = ["${data.aws_caller_identity.current.account_id}"]
```

A string with nothing except a single interpolated variable is not valid in Terraform anymore.

```suggestion
      values   = [data.aws_caller_identity.current.account_id]
```